### PR TITLE
Upgrade Cognite eslint config

### PR DESCRIPTION
**Pull request by halvard-cognite**
Now with typescript linting support :)

### CODE REVIEW

File: packages/react-scripts/package.json [hunk 1]
```diff
@@ -42,9 +42,9 @@
     "css-loader": "2.1.1",
     "dotenv": "6.2.0",
     "dotenv-expand": "4.2.0",
-    "eslint": "^6.1.0",
+    "eslint": "^5.3.0",
```
Comment (frikkf) [inline packages/react-scripts/package.json:45]: eslint": "^6.1.0" is the latest one. Could we go for that?

Comment (halvard-cognite) [reply packages/react-scripts/package.json:45]: Airbnb config that we are based on does not yet support it :/

**Comment by halvard-cognite** [comment]
Published a pre-release under `4.0.0-2`

File: packages/react-scripts/package.json (hunk 1 above)
Comment (halvard-cognite) [reply packages/react-scripts/package.json:45]: Also, quite a lot of the other packages specify 5* as dependency still

**Review by frikkf — APPROVED** [review]

**Pull request merged**